README.rst

```diff
@@ -25,7 +25,7 @@ Features of CrateDB:
 - Auto-partitioning, auto-sharding, and auto-replication.
 - Self-healing and auto-rebalancing.
 
-CrateDB offers the scalability and flexibility typically associated with a NoSQL database and is designed to run on inexpensive commodity servers and can be deployed and run across any sort of network. From personal computers to multi-region hybrid clouds.
+CrateDB offers the scalability and flexibility typically associated with a NoSQL database. CrateDB is designed to run on inexpensive commodity servers which can be deployed and run across any network from personal computers to multi-region hybrid clouds.
```

Comment:
this change repeats the word "CrateDB", which I would like to avoid. it also changes the semantics of the paragraph. "designed to run on inexpensive commodity servers" and "can be deployed and run across any sort of network" are two separate qualities. the "which" makes the second dependant on the first, which is inaccurate